src/main/java/uk/gov/hmcts/reform/civil/service/CoreCaseUserService.java

```diff
@@ -44,6 +44,19 @@ public void assignCase(String caseId, String userId, String organisationId, Case
         }
     }
 
+    public void unassignCase(String caseId, String userId, String organisationId, CaseRole caseRole) {
+        String caaAccessToken = getCaaAccessToken();
+        if (userWithCaseRoleExistsOnCase(caseId, caaAccessToken, caseRole)) {
+            CaseAssignedUserRoleWithOrganisation caseAssignedUserRoleWithOrganisation = CaseAssignedUserRoleWithOrganisation.builder()
```

Comment:
Does it make sense to use this type?
If someone is a LiP - they will not have an organisation.

Is there a seperate model / way to remove access for LiPs?
To me this seems like a way for removing Legal Rep Access - as they will be linked to an Org:

https://github.com/hmcts/civil-commons/blob/921ab4d407a60ca5ed877ec9dd1d289391814f9f/src/main/java/uk/gov/hmcts/reform/ccd/model/CaseAssignedUserRoleWithOrganisation.java#L14C1-L16C35

Reply:
if someone is lip they send a null organisation id. and it works fine. similar to when the case is assigned to a lip defendant

Reply:
Fair enough, perhaps this needs to be refactored in the future.
For now it seems to be flexing between LR and LIP